<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>880</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>623</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>2000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Qt AVR</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="cOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>1051</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>11</y>
      <width>127</width>
      <height>215</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="bBuild">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Build</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bFlash">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Flash</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bRun">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Microcontroller:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbController">
         <item>
          <property name="text">
           <string>AtMega32</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AtMega328</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AtTiny2313</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AtTiny4313</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>other</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Programer:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbFlashtool">
         <item>
          <property name="text">
           <string>AVR MKll</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>other</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lWD">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>50</y>
      <width>74</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Working dir:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>341</x>
      <y>91</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>c-Files:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="cCfiles">
    <property name="geometry">
     <rect>
      <x>341</x>
      <y>111</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextBrowserInteraction</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>331</x>
      <y>11</y>
      <width>198</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bAddFile">
       <property name="text">
        <string>Add c-File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bRemove">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>171</width>
      <height>216</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QPushButton" name="bNew">
       <property name="text">
        <string>New Project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bOpen">
       <property name="text">
        <string>Open Project</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>38</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>LF</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lLF">
             <property name="text">
              <string>0x62</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>HF</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lHF">
             <property name="text">
              <string>0xD9</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>EF</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lEF">
             <property name="text">
              <string>0xFF</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="bFuses">
         <property name="text">
          <string>Set Fuses</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="twEditor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>1051</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTextBrowser" name="tbEditor">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1041</width>
        <height>361</height>
       </rect>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9"/>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>cOutput</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>lWD</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>cCfiles</zorder>
   <zorder>groupBox</zorder>
   <zorder>twEditor</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
   </widget>
   <addaction name="menuData"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
